2.0/flask_app/services/vector_search.py

```
@@ -0,0 +1,91 @@
+
+    def _get_section_name_expr(self, table_name):
+        if table_name == 'exhibits':
+            return """
+            CASE 
+                WHEN s.exhibit_meaning IS NOT NULL 
+                    THEN s.exhibit_type || ' – ' || s.exhibit_meaning
+                ELSE s.exhibit_type
+            END
+            """
+        else:
+            return "s.section_name"
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)